package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@digitalbazaar/vc-commit-tracker",
+  "version": "0.0.1-0",
+  "description": "Retrieves the most recent commit dates from a list of VC specs and test suites.",
+  "type": "module",
+  "main": "lib/index.js",
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "scripts": {
+    "update-readme": "node ./lib/index.js",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "handlebars": "^4.7.8"
+  },
+  "devDependencies": {
+    "eslint": "^8.55.0",
+    "eslint-config-digitalbazaar": "^5.0.1",
+    "eslint-plugin-jsdoc": "^46.9.1",
+    "eslint-plugin-unicorn": "^49.0.0"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/vc-commit-tracker/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/vc-commit-tracker",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/vc-commit-tracker"
+  },
+  "directories": {
+    "lib": "./lib",
+    "templates": "./templates"
+  },
+  "engines": {
+    "node": ">=18"
```

Comment:
The build pipelines use v20, but I guess we know this to work in v18 also?

Reply:
It works in v18 too but for sake of consistency I've now updated `engines.node` to >= 20 [aa588e2](https://github.com/digitalbazaar/vc-commit-tracker/pull/1/commits/aa588e206c8e2ff697b84e12a00e1bc7a5a13645)